src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+  justify-content: space-between;
+  display: flex;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  width: 100%;
+  background-color: $main-background-color;
+
+  &__logo {
+    align-self: center;
+    margin-left: 50px;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
```

Comment:
Create separate file for this block styles

Reply:
Not fixed, nav is still in the header file 🤷